### PR TITLE
Limits Dogs from Lycancoughy virus

### PR DESCRIPTION
**Pull request by Aurorablade**
:cl:
tweak: Attemps to cap dogs you can cough up to 10..then you cough up a plush fox. Totally normal.
/:cl:

### CODE REVIEW

File: code/datums/diseases/lycancoughy.dm [hunk 1]
```diff
@@ -34,12 +35,20 @@
 												"<span class='userdanger'>You howl!</span>")
 				affected_mob.AdjustConfused(rand(6, 8))
 			if(prob(3))
-				var/list/puppytype = list(/mob/living/simple_animal/pet/corgi/puppy, /mob/living/simple_animal/pet/pug, /mob/living/simple_animal/pet/fox)
-				var/mob/living/puppypicked = pick(puppytype)
-				affected_mob.visible_message("<span class='danger'>[affected_mob] coughs up [initial(puppypicked.name)]!</span>", \
-													"<span class='userdanger'>You cough up [initial(puppypicked.name)]?!</span>")
+				if(barklimit <= 10)
```
Comment (Fox-McCloud) [inline code/datums/diseases/lycancoughy.dm]: This is still roughly 10*number of crew amount of animals on the station; this should probably just be a global variable attached to the disease and the limit increased to something like 20-30 (which is still a LOT).

File: code/datums/diseases/lycancoughy.dm [hunk 2]
```diff
@@ -34,12 +35,20 @@
...
+					var/list/puppytype = list(/mob/living/simple_animal/pet/corgi/puppy, /mob/living/simple_animal/pet/pug, /mob/living/simple_animal/pet/fox)
+					var/mob/living/puppypicked = pick(puppytype)
+					affected_mob.visible_message("<span class='danger'>[affected_mob] coughs up [initial(puppypicked.name)]!</span>", \
+														"<span class='userdanger'>You cough up [initial(puppypicked.name)]?!</span>")
+					new puppypicked(affected_mob.loc)
+					new puppypicked(affected_mob.loc)
+					barklimit ++
+				else
+					var/obj/item/toy/plushie/coughfox = /obj/item/toy/plushie/orange_fox
```
Comment (Fox-McCloud) [inline code/datums/diseases/lycancoughy.dm]: Why not spawn the corgi plushie as well?

This is still going to be an awful lot of objects created. Yeah, they won't be processing, but it's still going to really pile up horrifically.

File: code/datums/diseases/lycancoughy.dm (hunk 1 above)
Comment (Aurorablade) [reply code/datums/diseases/lycancoughy.dm]: how do i make it global for this in the proper manner not the 'fethas attempting to code' way.

File: code/datums/diseases/lycancoughy.dm (hunk 2 above)
Comment (Aurorablade) [reply code/datums/diseases/lycancoughy.dm]: I mean i could just leave it at the damage if you wanted.

File: code/datums/diseases/lycancoughy.dm [hunk 3]
```diff
@@ -34,12 +35,20 @@
...
+					new coughfox(affected_mob.loc)
+					affected_mob.visible_message("<span class='danger'>[affected_mob] coughs up a fox plushie!</span>", \
+														"<span class='userdanger'>You cough up FOX PLUSHIE?!</span>")
```
Comment (Tayyyyyyy) [inline code/datums/diseases/lycancoughy.dm]: Shouldn't this be "You cough up a FOX PLUSHIE?!"

Comment (Fox-McCloud) [reply code/datums/diseases/lycancoughy.dm]: Well, probably not. THe first message is what everyone else sees. The second is what the person themselves sees. I do think it'd be a *bit* more shocking to the person coughing up plushie than the people watching.

**Comment by Crazylemon64** [comment]
Max 10 per crewmember doesn't help very much at populations of 80+, but it's something

**Pull request merged**